components/NavBar/index.js

```diff
@@ -0,0 +1,83 @@
+import Link from "next/link";
+import styled from "styled-components";
+import { AiFillHome } from "react-icons/ai";
+import { useState } from "react";
+import { AiOutlineHome } from "react-icons/ai";
+import { RiCalendarTodoFill } from "react-icons/ri";
+import { RiCalendarTodoLine } from "react-icons/ri";
+import { BsFilePersonFill } from "react-icons/bs";
+import { BsFilePerson } from "react-icons/bs";
+
+export default function Navbar() {
+  const [home, setHome] = useState(false);
+  const [newTodo, setNewTodo] = useState(false);
+  const [about, setAbout] = useState(false);
```

Comment:
Not the most elegant way to solve navigation 😆 
A better solution would be to not use states at all. Instead, you'd link to different routes like `/activitiesForm`, the homepage and an about page. Within those routes you would check the current url and pass down its value to the `NavBar` component which in turn knows which icon to display depending on the value it receives.

Reply:
i knew this would be an issue but i was so proud that it works as desired 🤣😢
I'try to change it.